File: a2m-edu-Namtv/edu-mail/src/main/java/com/a2m/mail/shared/domain/SendMessageAction.java
package com.a2m.mail.shared.domain;

import com.a2m.mail.shared.data.SmtpMessageImpl;

public interface SendMessageAction {

	SmtpMessage getMessage();
    void setMessage(SmtpMessageImpl message);
    String getUsername();
    void setUsername(String username);
    String getReferences();
    String getInReplyTo();
    void setInReplyTo(String inReplyTo);
    void setReferences(String references);
    
}

File: a2m-edu-Namtv/edu-fe/src/app/model/mail/send-message-action.ts
import { SmtpMessage } from "./smtp-message";

export class SendMessageAction {
    message?: SmtpMessage = new SmtpMessage();
    username?: string = "";
    password?: String = "";
    constructor(msg?: SmtpMessage, username?: string, password?: string){
        this.message = msg;
        this.username = username;
        this.password = password
    }
}
